client/my-sites/promote-post-i2/components/post-item/style.scss

```diff
@@ -113,6 +113,8 @@ body.is-section-promote-post-i2 {
 
 		&__post-promote-button {
 			border-radius: 4px;
+			color: #3858ea;
```

Comment:
i think this is not quite right, we shouldnt hardcode the colors

have you tried to change your color palette and check out the results, (you can do so if you click the bottom right button and account settings) i guess that the colors should be shifting when changing 
![Screenshot 2024-01-24 at 18 03 13](https://github.com/Automattic/wp-calypso/assets/1416426/55f5f344-b3d9-43a9-b713-0610a752a32d)

like the one on the top right 


![Screenshot 2024-01-24 at 18 03 54](https://github.com/Automattic/wp-calypso/assets/1416426/1172b039-1890-4ace-b8b3-06f9dece5777)

Reply:
I saw Figma's file and thought we were gonna use the same color as there... so I should only outline the border and remove the background?

Reply:
I've fixed this behavior in my lates commit @sbarbosa  @therocket-gr